DUI3-DX/Sdk/Speckle.Connectors.Utils/Operations/SendOperation.cs

```diff
@@ -20,19 +20,24 @@ ISyncToMainThread syncToMainThread
     _syncToMainThread = syncToMainThread;
   }
 
-  public async Task<(string rootObjId, Dictionary<string, ObjectReference> convertedReferences)> Execute(
+  public async Task<(string rootObjId, Dictionary<string, ObjectReference> convertedReferences)?> Execute(
     IReadOnlyList<T> objects,
     SendInfo sendInfo,
     Action<string, double?>? onOperationProgressed = null,
     CancellationToken ct = default
   )
   {
-    Base commitObject = await _syncToMainThread
+    Base? commitObject = await _syncToMainThread
```

Comment:
I don't like these changes.
I don't think the builder should be returning null
If it is,  then the problem is with the builder, and we should fix this.

We should be using exceptions to communicate errors, A null return here gives us no context on what went wrong. We don't want silent failures.

Reply:
I think it's an artifact of Task usage that if the task throws an exception, the value ends up being null.

I need to double check things but I was definitely getting this when using the Rhino plugin and a conversion failed